tests/integration_tests/security/test_jail.py

```diff
@@ -514,25 +514,41 @@ def test_args_resource_limits(test_microvm_with_api):
     check_limits(pid, NOFILE, FSIZE)
 
 
-def test_negative_file_size_limit(uvm_plain):
+def test_positive_file_size_limit(uvm_plain):
     """
-    Test creating snapshot file fails when size exceeds `fsize` limit.
+    Test creating vm succeeds when memory size is under `fsize` limit.
+    This is caused by the fact that we back guest memory by memfd.
```

Comment:
looks like an out-of-date message in this test now

Reply:
removed